utils/dbfixup.py

```diff
@@ -87,7 +87,7 @@ def add_zero_bits(tile_type):
             print(line, file=f)
```

Comment:
Yes, I also think it would be beyond this ticket.

Which fuzzer exactly do you want to add this? There might be an easy 1:1 match for `CLBL?_?.SLICE?_X?.?FF.DMUX` and `CLBL?_?.SLICE?_X?.?MUX`, but the PIP stuff may be a bit harder. Most PIP patterns can be found by 050-intpips, but also by a more specialized later fuzzer. So we would need to replicate this in at least two places for each PIP. I don't see this getting any cleaner from that.